<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebDialogCreateProjectRefactor</class>
 <widget class="QWidget" name="WebDialogCreateProjectRefactor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>104</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="DialogLineEditEx" name="widget_title" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="DialogTextEditEx" name="widget_comment" native="true"/>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_codeWord">
     <property name="text">
      <string>Set code word</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DialogLineEditEx" name="widget_codeWord" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DialogLineEditEx</class>
   <extends>QWidget</extends>
   <header location="global">ui/controls/dialog_line_edit_ex.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DialogTextEditEx</class>
   <extends>QWidget</extends>
   <header location="global">ui/controls/dialog_text_edit_ex.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
